lib/impl/AzureBlobFiles.js

```diff
@@ -329,7 +334,23 @@ class AzureBlobFiles extends Files {
    * @private
    */
   _getUrl (filePath) {
-    return this._propsForPath(filePath).blockBlobURL.url.split('?')[0]
+    const azureURL = this._propsForPath(filePath).blockBlobURL.url.split('?')[0]
+    const index = azureURL.indexOf(AZURE_STORAGE_DOMAIN)
+    return 'https://' + DEFAULT_CDN_STORAGE_HOST + azureURL.substring(index + AZURE_STORAGE_DOMAIN.length)
+  }
+
+  /**
+   * @memberof AzureBlobFiles
+   * @override
+   * @private
+   */
+  async _getPresignUrl (filePath, options) {
+    if (!options || !options.expiryInSeconds) {
```

Comment:
options.blobName is also required

Reply:
blobName is not passed by users in options but as the filePath argument. We set blobName in next line as an option for TVM. So we won't need to check blobName requirement here.